main/index.js

```diff
@@ -66,12 +64,6 @@ process.env.STATION_BUILD_VERSION = BUILD_VERSION
 function handleError (/** @type {any} */ err) {
   Sentry.captureException(err)
 
-  ctx.recordActivity({
-    source: 'Station',
-    type: 'error',
-    message: `Station failed to start: ${err.message || err}`
-  })
```

Comment:
For simplicity, I gave the activity log all into `core`'s hands. I don't think the Desktop app _has_ to record extra activity.

Reply:
> IMO, we SHOULD log an activity item after we upgrade Core to a newer version. We want to let the user know that Station is still being actively developed, and they are receiving upgrades.

This can also be done from `core`: If it detects it was first launched with a new version (it can compare with state on disk), it can record this.

`core` has to have the activity logic, if we also add activity logic to `desktop` then things can get messy. What if their states get out of sync?